### PR TITLE
use local services for dev and test run

### PR DESCRIPTION
**Pull request by brianhatchl**

### CODE REVIEW

**Comment by sputnam** [comment]
@brianhatchl Will this require the use of a test setup script as well?

**Comment by brianhatchl** [comment]
oh yeah, it will.  just had the commits sitting around an wanted to push :grin:

**Comment by brianhatchl** [comment]
@maxgrossman I removed the `dist/` files that are now ignored by git, hence the many deletions.  This also fixes the karma tests to that they can run on a fresh, local hoot db / vm with no pre-existing data.

@sputnam I also fixed up the mocha tests, so we can now just run `npm test` to get the lint, mocha, and karma tests all running and passing.

To check for a passing Jenkins test see the main hoot PR that refs this branch https://github.com/ngageoint/hootenanny/pull/2916

File: test/hoot/spec/datasetPanel/tableBehavior.js [hunk 1]
```diff
@@ -36,11 +36,49 @@ module.exports = () => {
     }
 
     describe( 'table behavior', () => {
-        before( () => {
+        before( async function() {
```
Comment (maxgrossman) [inline test/hoot/spec/datasetPanel/tableBehavior.js]: next gen @brianhatchl

**Pull request merged**